### ISSUE TITLE
Increase the Issues to 100 from 50

### ISSUE THREAD

**Issue by ManuKun** [ManuKun/Excel_Exporter#3]

**Comment by AlithAnar** [comment]
This is already done in the main branch from what I can see

**Comment by ManuKun** [comment]
> This is already done in the main branch from what I can see

Yes! It was done by a Contributor Thank you Very Much for your Time!
